app/main/posts/views/mode-context-form-filter.directive.js

```diff
@@ -27,11 +27,25 @@ function ModeContextFormFilter($scope, FormEndpoint, PostEndpoint, TagEndpoint,
     $scope.changeForms = changeForms;
     activate();
 
+    $scope.$watch('filters', function () {
+            getPostStats(PostFilters.getFilters()).$promise.then(function (results) {
```

Comment:
Seems like the contents of `then` could move to function ie. `updateCounts(stats)` which would be called here and below in `activate`.. I'll poke at this and see if I can push this into todays release